src/vtcomposite.cpp

```diff
@@ -1000,14 +1185,30 @@ Napi::Value localize(Napi::CallbackInfo const& info)
         compress = comp_value.As<Napi::Boolean>().Value();
     }
 
+    // This if block must be validated *after* params.languages and params.worldviews
+    // because it checks return_localized_tile which is dictated by the
+    // value of both params.languages and params.worldviews.
+    if (return_localized_tile)
+    {
+        if (worldviews.empty())
+        {
+            worldviews.reserve(1);
```

Comment:
nit: no need to reserve 1

Reply:
Removed in 3711e05918f95ff3494302a0c36e07fe1bdfad08.